.github/rules/markdownlint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "default": true,
+    "MD013": {
+        "line_length": 400,
+        "heading_line_length": 100,
+        "code_block_line_length": 100,
+        "code_blocks": true,
+        "tables": true,
+        "headings": true,
+        "headers": true,
+        "strict": false,
+        "stern": false
+      },
+    "MD014": false,
+    "MD033": false,
+    "MD040": false
+}
```

Comment:
Please provide a small documentation why you choosed these settings.

Reply:
Where should I add the documentation since .json does not support comments?

Reply:
Create folder docs/ and put Markdown file in there.

Reply:
Done